### PR TITLE
Add clara/1.1.5 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **clara/1.1.5**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/clara/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import os
+from conans import ConanFile, tools
+
+
+class ZeroMQConan(ConanFile):
```
Comment (Croydon) [inline recipes/clara/all/conanfile.py]:
```suggestion
class ClaraConan(ConanFile):
```

Comment (madebr) [reply recipes/clara/all/conanfile.py]: Oops! Caught copy/pasting :)

**Comment by uilianries** [comment]
@madebr Please, take a look: https://github.com/madebr/conan-center-index/pull/5

File: recipes/clara/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    homepage = "https://github.com/catchorg/Clara"
+    topics = ("conan", "clara", "cli", "cpp11", "command-parser")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSL-1.0"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        self.output.warn("clara is unmaintained. Consider updating to lyra/1.2")
```
Comment (danimtb) [inline recipes/clara/all/conanfile.py]: Clara and Lyra are interchangeable?

Comment (madebr) [reply recipes/clara/all/conanfile.py]: Not interchangeable, but it is maintained.
There is a banner on clara's github page pointing to lyra.

Comment (uilianries) [reply recipes/clara/all/conanfile.py]: And we have Lyra in CCI, so it will not affect the users who are looking for Lyra after Clara.

Comment (danimtb) [reply recipes/clara/all/conanfile.py]: So if the project is nearly dead, why would someone use it? Are libraries relying on this one? Why did you *prioritize* creating this recipe and not others?

Comment (uilianries) [reply recipes/clara/all/conanfile.py]: fair enough, only for legacy propose I think.

**Pull request merged**